client/Components/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Typography, makeStyles, Button, Container } from '@material-ui/core';
```

Comment:
this is a nitpick but usually package imports come first, then local imports come second and you separate them with an empty line 
```js

import React from 'react';
...
// see how there's an empty line here?
import './LandingPage.scss';

```